hydrabus_framework/core/command/use.py

```diff
@@ -0,0 +1,16 @@
+def use(hbf_instance, command):
+    """
+    Method used to select a specific module
+    :param command: User input
+    :param hbf_instance: Hydrabus framework instance (self)
+    """
+    if len(command.split(" ")) < 2:
```

Comment:
As stated earlier this should be done by the dispatcher and should only check your varargs length

Reply:
Done [#01a8ef3](https://github.com/hydrabus-framework/framework/commit/01a8ef3a096bb762626bc71cba7fa939c6f3ffba)
In my opinion, i need to check the lenght and the veracity of the option exctract from user input command on command side. Indeed, if this is done on dispatcher side, I'm going to code several ugly elif.